CMake/cdat_modules/esmf_pkg.cmake

```diff
@@ -37,11 +37,11 @@ endif()
 if(CDAT_BUILD_ESMF_ESMP)
     if("${CMAKE_Fortran_COMPILER_ID}" STREQUAL GNU)
       # GNU gfortran must be >= 4.3
-      if(${Fortran_MAJOR_VERSION} GREATER 3 AND ${Fortran_MINOR_VERSION} GREATER 2)
+      if(${Fortran_MAJOR_VERSION} GREATER 3 AND ${Fortran_MINOR_VERSION} GREATER 1)
```

Comment:
I confirm that this problem was present also in Debian (in my case in the current testing version). I have changed as explained,

**CMake/cdat_modules/esmf_pkg.cmake**

From 
```bash
if(${Fortran_MAJOR_VERSION} GREATER 3 AND ${Fortran_MINOR_VERSION} GREATER 1)
```
to
```bash
if(${Fortran_MAJOR_VERSION} GREATER 4 AND ${Fortran_MINOR_VERSION} GREATER 0)
```
and the CMAKE seems to work.